### ISSUE TITLE
Successfull users-info request reported as errored

### ISSUE THREAD

**Issue by dianabarsan** [medic/cht-core#7642]
**Important**: This is a public repository. Anyone in the world can see what's posted here. If you are posting screenshots or log files, please **carefully examine them for** the presence of any kind of **protected health information** (PHI). Images or logs containing PHI _must_ be posted in fully-redacted form, with no visible PHI.

**Describe the bug**
When logging in as an offline user and watching the console, you'll see a warning: 
```
Error fetching users-info - ignoring {total_docs: 11455, warn_docs: 11455, warn: true, limit: 10000}
```
Even though the request was successful. 

**To Reproduce**
1. Observe browser console. 
2. Login as an offline user.
4. See warning

**Expected behavior**
There should be no warning. 

**Logs**
```
Error fetching users-info - ignoring {total_docs: 11455, warn_docs: 11455, warn: true, limit: 10000}
```

**Environment**
- Browser: Chrome
- App: webapp
- Version: 4.0

**Additional context**
Tagging this as low priority because the warning doesn't block execution and doesn't affect the rest of the workflow. https://github.com/medic/cht-core/blob/1f0f990150bf2e7df0242185f87365e96dbbecf9/webapp/src/js/bootstrapper/index.js#L69-L71

**Comment by dianabarsan** [comment]
@alium0978 Can we close this off if the fix was merged?

**Comment by dianabarsan** [comment]
It's not obvious if the fix was AT-ed.

**Comment by andrablaj** [comment]
No, I don't think it was AT-ed. That was my bad as I asked Samuel today if he merged the PR, rather than asking him if the PR was AT-ed. Sorry for that.
@tatilepizs @lorerod @ngaruko putting this on your radar when you perform release testing. Not sure what is the process when a PR gets merged without AT-ing, can you please advise on that? Thanks

**Comment by samuelimoisili** [comment]
Sorry, it wasn't AT-ed, how to I resolve it? Thanks for calling it out.

**Comment by tatilepizs** [comment]
Thank you @samuelimoisili and @andrablaj for the heads up.

---

### Test details

**Config:** Default
**Environment:** Local with docker helper script
**Platform:** WebApp
**Browser:** Chrome

### Reproducible on _cht old version_
I looked at the PRs that are still open to find one that was old enough so I can reproduce the error.
When I logged in with an offline user the warning **was shown.**

<details>
<summary>Image attached.</summary>

![image](https://user-images.githubusercontent.com/94494491/210273973-704e10ca-e2bf-4c5f-9e3f-b4e3e91de84f.png)

</details>

### Fixed on `master`

When I logged in with an offline user the warning **was not shown.**

<details>
<summary>Image attached.</summary>

![image](https://user-images.githubusercontent.com/94494491/210276311-cdad5f2c-b2f5-4472-9bf7-e17a80acf790.png)

</details>
